src/components/TodosFilter/TodosFilter.tsx

```diff
@@ -0,0 +1,83 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Status } from '../../types/Status';
+
+type Props = {
+  todos: Todo[];
+  filterBy: Status;
+  setFilterBy: React.Dispatch<React.SetStateAction<Status>>;
+  clearCompleted: () => void
+};
+
+export const TodosFilter: React.FC<Props> = (
+  {
+    todos,
+    filterBy,
+    setFilterBy,
+    clearCompleted,
+  },
+) => {
+  const handleSetFilteredTodos = (filter: Status) => () => {
```

Comment:
would be better to calls it like Filter or StatusFilter
```suggestion
  const handleSetFilteredTodos = (filter: Filter) => () => {
```